app/views/drafts/index.html.erb

```diff
@@ -2,8 +2,8 @@
   <section>
     <h2><%= current_user.provider_id %> Drafts</h2>
 
-    <% if @drafts.any? %>
-      <p><%= page_entries_info @drafts, entry_name: 'Draft' %></p>
+    <% if get_resources.any? %>
+      <p><%= page_entries_info get_resources, entry_name: 'Draft' %></p>
```

Comment:
This isnt used in this branch, however it IS used in MMT-998 so I'll make this change there.